index.ftd

```diff
@@ -1706,41 +1730,402 @@ width: fill-container
 
 
 
--- pr.toc-item list items: 
-$processor$: pr.toc
-
-- Section 1: basic-footer/
-  - Sub-section 1: /
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
-- Section 2: social-footer/
-  - Sub-section 2: /
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
-- Section 3: footer/
-  - Sub-section 3: /
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
-- Section 4: basic-footer/
-  - Sub-section 4:
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
-- Section 5: basic-footer/
-  - Sub-section 5:
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
\ No newline at end of file
+-- component post-card-ui:
+caption title:
+string timestamp:
+ftd.image-src image: 
+string url:
+body body:
+optional common.author-meta author:
+optional string tag:
```

Comment:
@saurabh-lohiya is this property also present in Meenu and Priyanka's set? ask them to add it.
Also, all the new properties in other components which are not in their sets ask Meenu and Priyanka to add and you also add their sets properties for all the components.